### PR TITLE
[Refactor] Refactor related log for tablet clone and scheduler

### PR DESCRIPTION
**Pull request by decster**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

Add and refactor some log related to tablet clone and scheduler

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 0 / 26 (00.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/clone/TabletScheduler.java|0|11|00.00%|
|:large_blue_circle:|com/starrocks/catalog/LocalTablet.java|0|6|00.00%|
|:large_blue_circle:|com/starrocks/master/ReportHandler.java|0|9|00.00%|

File: be/src/storage/snapshot_manager.cpp [hunk 1]
```diff
@@ -95,10 +94,18 @@ Status SnapshotManager::make_snapshot(const TSnapshotRequest& request, string* s
 
     StatusOr<std::string> res;
     if (request.__isset.missing_version) {
+        std::stringstream ss;
+        copy(request.missing_version.begin(), request.missing_version.end(), std::ostream_iterator<int64_t>(ss, ","));
+        LOG(INFO) << "make incremental snapshot tablet_id:" << request.tablet_id << " version:" << ss.str()
+                  << " timeout:" << timeout_s;
```
Comment (sduzh) [inline be/src/storage/snapshot_manager.cpp:100]: Have you considered using `JoinInts` in be/src/gutil/strings/join.h?

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 0 / 26 (00.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/clone/TabletScheduler.java|0|11|00.00%|
|:large_blue_circle:|com/starrocks/catalog/LocalTablet.java|0|6|00.00%|
|:large_blue_circle:|com/starrocks/master/ReportHandler.java|0|9|00.00%|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 0 / 26 (00.00%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/clone/TabletScheduler.java|0|11|00.00%|
|:large_blue_circle:|com/starrocks/catalog/LocalTablet.java|0|6|00.00%|
|:large_blue_circle:|com/starrocks/master/ReportHandler.java|0|9|00.00%|

**Review by chaoyli — APPROVED** [review]

**Review by Astralidea — APPROVED** [review]

**Pull request merged**